scripts/storage-emulator-integration/tests.ts

```diff
@@ -1989,11 +1989,135 @@ describe("Storage emulator", () => {
       if (!TEST_CONFIG.keepBrowserOpen) {
         await browser.close();
       }
-      if (!TEST_CONFIG.useProductionServers) {
-        await test.stopEmulators();
+      if (TEST_CONFIG.useProductionServers) {
+        delete process.env.GOOGLE_APPLICATION_CREDENTIALS;
       } else {
+        await test.stopEmulators();
+      }
+    });
+  });
+
+  emulatorSpecificDescribe("Emulator Integrity", function (this) {
```

Comment:
Not sure I understand the test name

Reply:
Renamed to Emulator Internals, what do you think?